tiledb/sm/array/array.cc

```diff
@@ -506,6 +536,11 @@ Status Array::set_timestamp(uint64_t timestamp) {
   return Status::Ok();
 }
 
+Status Array::set_config(const Config& config) {
+  config_ = config;
```

Comment:
Make sure that if no config is set, then the context config should be inherited.

Reply:
A unit test for just that should be added as well.